geckomat/limit_proteins/updateProtPool.m

```diff
@@ -1,37 +1,39 @@
-function model  = updateProtPool(model,Ptot,fs)
+function model  = updateProtPool(model,Ptot,Pmeas,f,sigma)
 %Calculate total mass of bounded enzymes
 if any(isnan(model.concs))
     poolIndex  = find(strcmpi(model.rxns,'prot_pool_exchange'));
-    Pmeasured  = sum(model.concs(~isnan(model.concs)));
-    difference = Ptot-Pmeasured;
+    difference = Ptot-Pmeas;
```

Comment:
@edkerk your suggestions have been addresses in commit 364cb2e, which also adds some extra information regarding the different cases for `difference`.

Reply:
> Your strategy is indeed more correct.
> 
> Two suggestions that I could not directly comment in the code:
> 
> * Should line 93 and 99 not use `Ptot` instead of `Pbase`, as the protein content from the model does not take into account previous flexibilization?
> * In `generate_protModels`, should the `f` used in `constrainEnzymes` not be calculated from the flexibilized `abundances`, instead of based on the "raw" data from `load_Prot_Ferm_Data`?
> 
> Regardless, the changes now break the models that I work with (`newPool = (difference)*fs` in `updateProtPool` rescued them, but this line is obviously wrong I now see). I'm becoming less convinced about using condition-specific `f` calculations, will explain that in #130 (as it is not _really_ the topic of this PR). The changes here should be implemented regardless though.

@edkerk I agree with the second point, it is correct to calculate the `f-factor` after flexibilizing the data. I've fixed this with commit aee2f3c. Regarding your first point are you referring to lines 93 and 99 in the `constrainEnzymes` function?

Reply:
@IVANDOMENZAIN , yes in `constrainEnzymes`.

Reply:
Ok, gonna check it

Reply:
@edkerk I've addressed your first point (regarding Ptot and Pbase)in commit fabe6aa, I agree with your comment. In this new version the protein content of the model is used just for the case in which no `Ptot` is provided as input or in model parameters. I've run the test case again and no changes were observed in any output files. Ready to merge now, I think.

Reply:
@IVANDOMENZAIN Excellent, I agree with your changes, also no changes in the models here. I leave it up to you to merge :)